core/Tracker/LogTable.php

```diff
@@ -82,4 +82,8 @@ public function getLinkTableToBeAbleToJoinOnVisit()
         return;
     }
 
+    public function getPrimaryKey()
+    {
+        return array();
```

Comment:
We can use this information also later to automatically generate a log table when a plugin defines a `LogTable` and then fully automatically add all dimensions /columns that belong to that table and automatically start tracking data into it. It is for now only vision but those LogTable classes can be nicely auto generated in the future 👍